### DISCUSSION TITLE
Add us? We're SIA...

### DISCUSSION THREAD

**Discussion by siamelanies** [code4recovery/12-step-meeting-list#1196] [Ideas]
Hi, just found this plugin while trying to revamp a site for Survivors of Incest, Anonymous at siawso.org. We've been around for almost 41 years. In the settings, there's a dropdown list of recovery programs - can we be added? 
Thanks so much, this plugin has been a lifesaver. :-)
Melanie S

**Comment by joshreisner** [comment]
hi @siamelanies sure we'd be happy to

what meeting types do you have? for an example, here are the existing AA meeting types: https://github.com/code4recovery/spec#meeting-types

**Reply by siamelanies** [reply]
Sweet, thanks, @joshreisner !

Most meetings we have are are a subset of the AA types (LGBTQ, POC, Women, Men, newcomer,
12x12, Step Study) 

What I've not found in the list:
Afrikaans Language
Dutch Language
Arabic Language
Slovenian Language
Thai Language
Turkish Language
Steps Workshop (different to Step Study)
Open Share Format (different to Open meeting)
Genderqueer (possibly different to Transgender)

I think that's it for meeting types rn. Anything else I need to do?

**Reply by joshreisner** [reply]
> Most meetings we have are are a subset of the AA types (LGBTQ, POC, Women, Men, newcomer,
12x12, Step Study)

if you could specify the subset of types you want that would help us move forward - or did you want them all?

11th Step Meditation
12 Steps & 12 Traditions
Afrikaans
American Sign Language
Arabic
As Bill Sees It
Babysitting Available
Big Book
Birthday
Bisexual
Breakfast
Candlelight
Child-Friendly
Closed
Concurrent with Al-Anon
Concurrent with Alateen
Cross Talk Permitted
Daily Reflections
Digital Basket
Discussion
Dual Diagnosis
Dutch
English
Fragrance Free
French
Gay
Genderqueer
Grapevine
Hebrew
Indigenous
Italian
Japanese
Korean
Lesbian
LGBTQ
Literature
Living Sober
Location Temporarily Closed
Meditation
Men
Native American
Newcomer
Open
Open Share Format
Outdoor
People of Color
Polish
Portuguese
Professionals
Punjabi
Russian
Secular
Seniors
Slovenian
Smoking Permitted
Spanish
Speaker
Step Study
Steps Workshop
Thai
Tradition Study
Transgender
Turkish
Wheelchair Access
Wheelchair-Accessible Bathroom
Women
Young People

**Reply by siamelanies** [reply]
Here you go (I almost forgot about Finnish and Ukranian).  Thanks again!

12 Steps & 12 Traditions
Afrikaans
American Sign Language
Arabic
Big Book
Closed
Dutch
English
Finnish
Fragrance Free
French
Genderqueer
Hebrew
Indigenous
Italian
Japanese
Korean
LGBTQ
Literature
Location Temporarily Closed
Men
Native American
Newcomer
Open
Open Share Format
People of Color
Polish
Portuguese
Punjabi
Russian
Slovenian
Spanish
Speaker
Step Study
Steps Workshop
Thai
Transgender
Turkish
Ukranian
Wheelchair Access
Wheelchair-Accessible Bathroom
Women

**Reply by joshreisner** [reply]
hi @siamelanies we've released `4.16.20` with a program option for SIA

we understand that "Persian" is the preferred term so we added it that way, but if you would like to switch it to Farsi just add this code to your theme's `functions.php` file:

```
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'FA' => 'Farsi',
    ));
}
```

**Comment by siamelanies** [comment]
And one more: Ritual-Abuse
thanks!

**Comment by siamelanies** [comment]
And Farsi. Thanks for your patience: Here's the new list,

12 Steps & 12 Traditions
Afrikaans
American Sign Language
Arabic
Big Book
Closed
Dutch
English
Farsi
Finnish
Fragrance Free
French
Genderqueer
Hebrew
Indigenous
Italian
Japanese
Korean
LGBTQ
Literature
Location Temporarily Closed
Men
Native American
Newcomer
Open
Open Share Format
People of Color
Polish
Portuguese
Punjabi
Ritual Abuse
Russian
Slovenian
Spanish
Speaker
Step Study
Steps Workshop
Thai
Transgender
Turkish
Ukranian
Wheelchair Access
Wheelchair-Accessible Bathroom
Women

**Comment by siamelanies** [comment]
Ooh, great thanks, and you're right, we'll use Persian.
